pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -128,7 +128,7 @@ func RunNewBuild(fullName string, f *clientcmd.Factory, out io.Writer, in io.Rea
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "A build configuration was created - you can run `%s start-build %s` to start it.\n", fullName, t.Name)
+			fmt.Fprintf(c.Out(), "A build configuration was created and an initial build triggered - you can run `%s build-logs %s-1` to stream the logs as the build progresses.\n", fullName, t.Name)
```

Comment:
Recreating the build with oc new-build starts from -1 again.
I didn't find a way to get the build name without an API call. If this is a concern, then perhaps we can be more verbose and suggest oc get builds, oc build-logs BUILD-NAME.